develop/dev-guide-proxysql-integration.md

```diff
@@ -104,579 +655,785 @@ The production environment requires more steps than the test environment. To dep
+    ```bash
     systemctl start proxysql
     ```
 
-### Other installation ways
+To learn more about the supported platforms of ProxySQL and their installation, refer to [ProxySQL README](https://github.com/sysown/proxysql#installation) or [ProxySQL installation documentation](https://proxysql.com/documentation/installing-proxysql/).
 
-To install ProxySQL using other ways, refer to the [ProxySQL README](https://github.com/sysown/proxysql#installation) or the [ProxySQL installation documentation](https://proxysql.com/documentation/).
+#### Step 3. Configure ProxySQL
 
-## 3. Configure ProxySQL
+To use ProxySQL as a proxy for TiDB, you need to configure ProxySQL. To do so, you can either execute SQL statements inside ProxySQL Admin Interface (recommended) or use the [configuration file](#TODO).
 
-To use ProxySQL as a proxy for TiDB, you need to configure ProxySQL. The required configuration items are listed in the following sections. For more details about other configuration items, refer to the [ProxySQL official documentation](https://proxysql.com/documentation/).
+> **Note:**
+>
+> The following sections list only the required configuration items.
+> For a comprehensive list of configurations, see [ProxySQL documentation](https://proxysql.com/documentation/proxysql-configuration/).
 
-### Simple introduction
+##### Option 1: Configuring ProxySQL using the Admin Interface
 
-ProxySQL uses a port to manage configuration, which is **_ProxySQL Admin interface_**, and a port to proxy, which is **_ProxySQL MySQL Interface_**.
+1. Reconfigure ProxySQL’s internals using the standard ProxySQL the Admin interface, accessible via any MySQL command line client (available by default on port `6032`):
 
-- **_ProxySQL Admin interface_**: To connect to the admin interface, you can use an `admin` user to read and write configuration, or use a `stats` user to read part of statistics (cannot read or write configuration). The default credentials are `admin:admin` and `stats:stats`. For security reasons, you can use the default credentials to connect locally, but to connect remotely, you need to configure a new user, which is often named `radmin`.
-- **_ProxySQL MySQL Interface_**: Used as a proxy to forward SQL to the configured service.
+    ```bash
+    mysql -u admin -padmin -h 127.0.0.1 -P6032 --prompt 'ProxySQL Admin> '
+    ```
 
-![proxysql config flow](/media/develop/proxysql_config_flow.png)
+    The above step will take you to the ProxySQL admin prompt.
 
-There are three layers in ProxySQL configurations: `runtime`, `memory`, and `disk`. You can change the configuration of the `memory` layer only. After modifying the configuration, you can use `LOAD xxx TO runtime` to make the configuration effective, and/or you can use `SAVE xxx TO DISK` to save to the disk to prevent configuration loss.
+2. Configure the TiDB servers to be used, where you can add one or multiple TiDB servers to ProxySQL. In this example, one TiDB server is added as follows:
```

Comment:
@Icemap replace this line 
Configure the TiDB servers to be used, where you can add one or multiple TiDB servers to ProxySQL. In this example, one TiDB server is added as follows:
**WITH**
Configure the TiDB servers to be used, you can add one or multiple TiDB servers to ProxySQL. In this example we will be add TiDBCloud instance. Replace '<tidb cloud dedicated cluster host>' and <tidb cloud dedicated cluster port> with the actual host and port that's been configured.